CHANGES.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 0.10.4 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Documentation.
```

Comment:
Don't need changelog for this. We don't need to cut a release for documentation to be updated, and in this case we haven't even touched the docstrings.